src/utilities/timeline.ts

```diff
@@ -334,9 +334,9 @@ export function createRow(timelines: Timeline[]): Row {
   const id = getNextRowID(timelines);
 
   return {
-    autoAdjustHeight: true,
+    autoAdjustHeight: false,
     expanded: true,
-    height: 200,
+    height: 400,
```

Comment:
Can we keep the default to `200` just so it looks like there is more space for additional rows? Future: add some kind of "+ new row" at the bottom that takes you to the timeline editor to create a new row.

Reply:
Yeah can do, does mean you'd have to expand the initial activity row to see anything but a small amount of activities without triggering overdraw but that's probably fine? May be better than making it look like there's no more space for rows.

Reply:
The "+ new row" button would be great